chart/values.yaml

```diff
@@ -183,6 +183,50 @@ ingress:
     # Http paths to add to the web Ingress after the default path
     succeedingPaths: []
 
+  # Configs for the Ingress of the RPC server Service
```

Comment:
I don't think we need or want ingress for this component. The service should be sufficient.